api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/TXTBCohortQueries.java

```diff
@@ -541,7 +541,7 @@ public CohortDefinition getPatientsWithAtLeastPosNegInvestigationResultTB() {
             + "    o.voided = 0 AND "
             + "    e.encounter_type IN (${adultoSeguimentoEncounterType},${pediatriaSeguimentoEncounterType})  AND "
             + "    o.concept_id = ${researchResultConcept} AND "
-            + "    o.value_coded IN (${positiveConcept},${negativeConcept}) AND "
+            + "    o.value_coded = ${positiveConcept}AND "
             + "    e.encounter_datetime BETWEEN :startDate AND :endDate AND "
             + "    e.location_id  = :location "
             + "GROUP BY p.patient_id;";
```

Comment:
I don't think we need that semi colon(;) inside the string then have another one outside

Reply:
Should i remove for all occurences on this file?

Reply:
Done

Reply:
That semi colon inside the sring is NOT required, quickly test after removing those